app/models/like.rb

```diff
@@ -5,4 +5,5 @@ class Like < ApplicationRecord
   scope :already_liked, (lambda do |user_id, review_id|
     where user_id: user_id, review_id: review_id
   end)
+  scope :user_review, ->(review_id){where review_id: review_id}
```

Comment:
scope ko mapping gì với xử lý hết thanh niên

em đang lấy ra toàn bộ like của review mà

sửa lại tên scope hì

Reply:
scope này em viết để kiểm tra xem user đã like review hay chưa đó chị

Reply:
ukm, nhưng đọc tên scope chị lại ko hiểu dc mục đích của em như thế

ngoài ra, cái này là truyền vào id_review ở model like tức là lấy ra like của review có id mà em truyền vào mà, có nhầm nhọt gì ko em